packages/xo-web/src/common/xo/index.js

```diff
@@ -965,28 +968,40 @@ export const startVm = async (vm, host) => {
 
   const id = resolveId(vm)
   const hostId = resolveId(host)
-  return _call('vm.start', {
-    id,
-    host: hostId,
-  }).catch(async reason => {
-    if (!forbiddenOperation.is(reason)) {
+  return _startVm(id, hostId).catch(async reason => {
+    if (!forbiddenOperation.is(reason) && !operationFailed.is(reason)) {
       throw reason
     }
 
-    const choice = await chooseActionToUnblockForbiddenStartVm({
-      body: _('blockedStartVmModalMessage'),
-      title: _('forceStartVmModalTitle'),
-    })
-
-    if (choice === 'clone') {
-      return cloneAndStartVm(vm, host)
+    if (reason.data.code === 'DUPLICATED_MAC_ADDRESS') {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmWithDuplicatedMacAddressesMessage'),
+        })
+        await _startVm(id, hostId, { bypassMacAddressesCheck: true })
+      } catch (error) {
+        if (error === undefined || !forbiddenOperation.is(error)) {
+          throw error
+        }
+        reason = error
+      }
```

Comment:
Instead of testing `undefined`, you can make the `try/catch` start after `startVm`:
```suggestion
      await confirm({
        title: _('forceStartVm'),
        body: _('vmWithDuplicatedMacAddressesMessage'),
      })
      try {
        await _startVm(id, hostId, { bypassMacAddressesCheck: true })
      } catch (error) {
        if (!forbiddenOperation.is(error)) {
          throw error
        }
        reason = error
      }
```